packages/transformers/babel/src/config.js

```diff
@@ -59,6 +60,16 @@ export async function load(
     return;
   }
 
+  const babelCore: BabelCore = await options.packageManager.require(
```

Comment:
This won't get tracked for cache invalidation. We need to call `config.addDevDependency` here as well.